security/imx_ahab/sign-file.sh

```diff
@@ -122,6 +128,14 @@ fi
 # working file used for signature
 cp ${WORK_FILE} ${WORK_FILE}.mod
 
+# enable revokation
+# info from CST_UG, section “B SRK revocation on HABv4”, page 18
```

Comment:
not sure I would like to mention all these details, maybe just the section name is enough (if AN gets updated in the future)
```suggestion
# info from CST_UG, section “B SRK revocation on HABv4”, page 18
```